src/game/logic/object/armortemplateset.h

```diff
@@ -40,7 +40,12 @@ enum ArmorSetType
 class ArmorTemplateSet
 {
 public:
+    ArmorTemplateSet() { Clear(); }
+    void Clear();
     void Parse_Armor_Template_Set(INI *ini);
+    const BitFlags<ARMORSET_COUNT> &Get_Conditions_Yes(int unk) const { return m_conditions; }
+    int Unk() const { return 1; }
+    Utf8String Get_Definition() const { return "WeaponTemplateSet"; }
```

Comment:
This should return "ArmorTemplateSet", no?

Reply:
fixed.